### PR TITLE
Improve test coverage for TFE workers

### PR DESCRIPTION
**Pull request by mortendahl**
Builds on https://github.com/OpenMined/PySyft/pull/2247.
Closes https://github.com/OpenMined/PySyft/issues/2242.

### CODE REVIEW

File: examples/tutorials/Part 13b - Secure Classification with Syft Keras and TFE - Secure Model Serving.ipynb [hunk 1]
```diff
@@ -138,22 +138,41 @@
...
    "cell_type": "code",
    "execution_count": 5,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "INFO:tf_encrypted:If not done already, please launch the following command in a terminal on host 'localhost:4000':\n",
```
Comment (robert-wagner) [inline examples/tutorials/Part 13b - Secure Classification with Syft Keras and TFE - Secure Model Serving.ipynb]: Should this be here?

File: syft/workers/tfe.py [hunk 2]
```diff
@@ -18,14 +18,13 @@ def __init__(self, host=None, auto_managed=True):
         self._server_process = None
         self._auto_managed = auto_managed
 
-    def start(self, player_name, *workers):
+    def start(self, player_name, cluster):
```
Comment (midokura-silvia) [inline syft/workers/tfe.py]: Could you please add docstrings to the functions.

File: examples/tutorials/Part 13b - Secure Classification with Syft Keras and TFE - Secure Model Serving.ipynb (hunk 1 above)
Comment (mortendahl) [reply examples/tutorials/Part 13b - Secure Classification with Syft Keras and TFE - Secure Model Serving.ipynb]: [The existing notebook](https://github.com/OpenMined/PySyft/blob/dev/examples/tutorials/Part%2013b%20-%20Secure%20Classification%20with%20Syft%20Keras%20and%20TFE%20-%20Secure%20Model%20Serving.ipynb) has the same, caused by `AUTO = False`. I'm good either way (leaning towards `AUTO = True`) but can't remember if @iamtrask had his reasons..?

**Comment by mortendahl** [comment]
Moved changes to https://github.com/OpenMined/PySyft/pull/2247

**Pull request closed without merging**